mozdownload/__init__.py

```diff
@@ -2,4 +2,15 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from scraper import *
+from scraper import PLATFORM_FRAGMENTS, NotSupportedError, NotFoundError, \
+    NotImplementedError, TimeoutError, Scraper, DailyScraper, \
+    DirectScraper, ReleaseScraper, ReleaseCandidateScraper, \
+    TinderboxScraper, TryScraper
```

Comment:
Tests should import this constant via `from mozdownload.scraper import PLATFORM_FRAGMENTS`. That will still be allowed.